server/events/command_runner.go

```diff
@@ -601,6 +651,11 @@ func (c *DefaultCommandRunner) parallelPlanEnabled(ctx *CommandContext, projectC
 	return len(projectCmds) > 0 && projectCmds[0].ParallelPlanEnabled
 }
 
+// parallelPolicyCheckEnabled returns true if parallel plan is enabled in this context.
+func (c *DefaultCommandRunner) parallelPolicyCheckEnabled(ctx *CommandContext, projectCmds []models.ProjectCommandContext) bool {
```

Comment:
Should we just only support this, instead of also supporting the flag? Doesn't seem like anyone would care about non-parallel policy checks when there are parallel policy plans or vice versa